src/tests/functional/plugin/shared/include/snippets/two_inputs_and_outputs.hpp

```diff
@@ -26,6 +26,11 @@ class TwoInputsAndOutputs : public testing::WithParamInterface<ov::test::snippet
     void SetUp() override;
 };
 
+class TwoInputsAndOutputsWithReversedOutputs : public TwoInputsAndOutputs {
```

Comment:
Honestly it'd be great if you add the comment what is verified by this test (why we add almost the same test as above), smth like: The `Result` ops have the specific order in topological sorting. The test checks the correctness of the algorithm for checking possible cyclic dependency in tokenization.

Because after month we'll forget why we have almost the same tests, what is the difference between them :smile:

Reply:
done